### PR TITLE
Add blogpost announcing our GSoC projects 2020

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: pages/news/2020-05-05-mixxx-gsoc-projects-2020.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+date: 2020-05-05 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+As you may have read, Mixxx participates in this year's Google Summer of Code, an opportunity for students to work on Mixxx full-time as a 3-month summer project and receive a stipend from Google.
```
Comment (Be-ing) [inline pages/news/2020-05-05-mixxx-gsoc-projects-2020.html]:
```suggestion
Mixxx is participating in this year's [Google Summer of Code](https://summerofcode.withgoogle.com/), an opportunity for students to work on Mixxx full-time as a 3-month summer project and receive a stipend from Google.
```

**Comment by Holzhaus** [comment]
It looks a bit strange because we use the full names for Harshit and Cristiano, but not for Janek. I messaged Janek on Zulip and asked if it's okay to use his full name in the announcement.

**Comment by Holzhaus** [comment]
I think this is ready. Merge?

**Comment by Be-ing** [comment]
Waiting for CI

**Comment by Be-ing** [comment]
:( CI won't go because build-ubuntu-18.04-amd64-rryan is offline.

File: pages/news/2020-05-05-mixxx-gsoc-projects-2020.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+[Uwe Klotz](https://github.com/uklotzde) will be his primary mentor.
+
+Janek will be working on a [macro recording feature](https://summerofcode.withgoogle.com/projects/#4641898733502464) inspired by [Serato Flip](https://serato.com/dj/pro/expansions/flip) to record custom edits in Mixxx without altering the audio files.
+His project will be mentored by [Jan Holthuis](https://github.com/Holzhaus).
+
+We're going to do things a bit differently for GSoC this year than we have in the past.
+Before, mentors have had a lot of private one-on-one communication with students.
+This year we'll be encouraging students to discuss their work more publicly so the whole community is involved throughout their projects.
+
+Thus, if you're interested in these projects check our [development](https://mixxx.zulipchat.com/) for discussions and progress reports.
```
Comment (Be-ing) [inline pages/news/2020-05-05-mixxx-gsoc-projects-2020.html]:
```suggestion
Thus, if you're interested in these projects check our [Zulip chat](https://mixxx.zulipchat.com/) for discussions and progress reports.
```

Comment (Holzhaus) [reply pages/news/2020-05-05-mixxx-gsoc-projects-2020.html]: development chat?

**Comment by Holzhaus** [comment]
Ping @rryan

File: pages/news/2020-05-05-mixxx-gsoc-projects-2020.html (hunk 2 above)
Comment (uklotzde) [reply pages/news/2020-05-05-mixxx-gsoc-projects-2020.html]: Or "developers chat"? But the chat is not only for developers, also for users.

Comment (Holzhaus) [reply pages/news/2020-05-05-mixxx-gsoc-projects-2020.html]: Just "chat" then?

Comment (Be-ing) [reply pages/news/2020-05-05-mixxx-gsoc-projects-2020.html]: Why not "Zulip chat"?

Comment (Holzhaus) [reply pages/news/2020-05-05-mixxx-gsoc-projects-2020.html]: Ok

**Comment by Be-ing** [comment]
I'll merge this now, but someone needs to manually trigger a build when the build server is back online.

**Pull request merged**